haskell/marlowe.cabal

```diff
@@ -1,4 +1,4 @@
-name:                   marlowe
+name:                   marlowe-reference
```

Comment:
I renamed the `reference` project to this so that the exported project could be called `marlowe`